.travis.yml

```diff
@@ -1,48 +1,25 @@
-language: go
-go:
-  - 1.5.1
-env:
-  - GO15VENDOREXPERIMENT=1
+language: generic
 branches:
   only:
     - master
+env:
+  DEV_REGISTRY=quay.io
```

Comment:
@mboersma good catch - removed in https://github.com/arschles/deis-workflow-cli/commit/3c52ee0011121d07831cc007faaf170b86bec1a5